Simulation/Models/GravityModel_Generator.m

```diff
@@ -0,0 +1,10 @@
+function Gen_GravityModel = GravityModel_Generator(Magnitude,Direction)
+
+  Gen_GravityModel = @GravityModel;

  
```

Comment:
Same thing about CRLF vs LF goes here too.

Reply:
Made the file LF explicitly